src/Test/Utilities/Shared/Compilation/OperationTreeVerifier.cs

```diff
@@ -208,7 +208,8 @@ private void VisitInstanceExpression(IOperation instance)
 
         internal override void VisitNoneOperation(IOperation operation)
         {
-            Assert.True(false, "Encountered an IOperation with `Kind == OperationKind.None` while walking the operation tree.");
```

Comment:
Correct.  We have some cases where bound nodes don't get converted to proper operations.  I wanted to at least still allow baselines to get generated, rather than having tests that coudln't actually verify anything.  We can trivially change this back in the future once we're reasonably sure that we've completed converting all bound nodes to operations.